python/requirements.txt

```diff
@@ -22,7 +22,7 @@ durationpy==0.5
     # via -r requirements.in
 expiringdict==1.2.2
     # via -r requirements.in
-flask==2.2.2
+flask==2.2.5
```

Comment:
I would recommend just bumping it to latest version which is 2.3.2 as seen here: https://flask.palletsprojects.com/en/2.3.x/changes/

This should bump the `Werkzeug` lib as well which should quiet scanners down.